src/dataflow/src/server.rs

```diff
@@ -522,41 +522,33 @@ where
         updates.clear();
     }
 
+    // TODO eli: this is where interesting frontier logging happens
```

Comment:
This seems like a weird TODO. Intentional?

Reply:
Not intentional, a ‘finding my way around’ comment I forgot to delete.